cityofaustin/atd-moped
# Remove funding_source_name from AGOL database view and ETL

## Associated issues
Closes https://github.com/cityofaustin/atd-data-tech/issues/19023

This PR removes a column from the AGOL database view and the ETL that transfers its value to the AGOL dataset since it is no longer used in the ATSD AGOL app.

## Testing
**URL to test:** 

Local

**Steps to test:**
1. Start your local stack with or without production data
2. Explore your DB with your favorite client and see that the `component_arcgis_online_view` DB view no longer selects `funding_source_name` from the joined `project_list_view`
3. Run the down migration and see that `funding_source_name` is back

**Note**: I also removed `funding_source_name` from the GraphQL query used by the AGOL ETL. This will not break the ETL, and, after the release next Thursday, the ETL will no longer populate this field in the AGOL dataset. I will then request for this field to be removed from the AGOL dataset in https://github.com/cityofaustin/atd-data-tech/issues/17971. Doing this out of order would break the ETL since AGOL's API would complain about not having a matching field in the AGOL dataset during the data transfer.

---
#### Ship list
